app/javascript/components/App.js

```diff
@@ -27,10 +27,46 @@ export default class App extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-      books: MockBooks
+      books: [],
+      rentals: []
     }
   }
 
+  componentDidMount() {
+    this.bookIndex()
+    this.rentalIndex()
+  }
+
+  bookIndex = () => {
+    fetch("http://localhost:3000/books")
```

Comment:
So that this works on Heroku, remove the `http://localhost:3000` so its just `/books`, all requests should still work just fine

Reply:
Yeah we caught this on the current branch we're working on and changed it. So I'm going to leave them in this branch and they'll be fixed in the next. Thank you!